types/gestalt/index.d.ts

```diff
@@ -1526,7 +1531,12 @@ export interface VideoProps {
     onSeek?: AbstractEventHandler<React.SyntheticEvent<HTMLVideoElement>>;
     onTimeChange?: AbstractEventHandler<React.SyntheticEvent<HTMLVideoElement>, { time: number }>;
     onVolumeChange?: AbstractEventHandler<React.SyntheticEvent<HTMLDivElement>, { volume: number }>;
-
+    onError?: AbstractEventHandler<React.SyntheticEvent<HTMLDivElement> | React.SyntheticEvent<HTMLAnchorElement>>;
+    onLoadStart?: AbstractEventHandler<React.SyntheticEvent<HTMLDivElement> | React.SyntheticEvent<HTMLAnchorElement>>;
+    onPlaying?: AbstractEventHandler<React.SyntheticEvent<HTMLDivElement> | React.SyntheticEvent<HTMLAnchorElement>>;
+    onSeeking?: AbstractEventHandler<React.SyntheticEvent<HTMLDivElement> | React.SyntheticEvent<HTMLAnchorElement>>;
+    onStalled?: AbstractEventHandler<React.SyntheticEvent<HTMLDivElement> | React.SyntheticEvent<HTMLAnchorElement>>;
+    onWaiting?: AbstractEventHandler<React.SyntheticEvent<HTMLDivElement> | React.SyntheticEvent<HTMLAnchorElement>>;
```

Comment:
Some of the types are supposed to be `HTMLVideoElement` instead of `HTMLDivElement`. Also `HTMLAnchorElement` doesn't seem correct for some of these callbacks.

See https://github.com/pinterest/gestalt/blob/master/packages/gestalt/src/Video.js#L43

Reply:
oh my bad, I made the cjanges based on these types https://github.com/pinterest/gestalt/pull/1515/commits , but didnt look down the commit where the person actullay changed it to video element, thanks for reviewing